src/storefront/src/templates/partials/_footer.pug

```diff
@@ -62,7 +62,7 @@ section.uk-section.uk-section-secondary.uk-section-xsmall.uk-light
         div.uk-margin-small.uk-text-muted.uk-text-small
           |Made with
           +icon('heart', '0.75')(class="uk-margin-xsmall-left uk-margin-xsmall-right")
-          |by Oracle A-Team
+          |by Oracle A-Team Alumni
```

Comment:
I like it... laughed... but don't think we should do it

Reply:
But... but... the badges 🤣

Reply:
:+1

Reply:
After voting, we will keep like this for now. Can re-visit later.